android/assets/jsons/Civ V - Gods & Kings/Units.json

```diff
@@ -1194,7 +1194,7 @@
 		"requiredResource": "Oil",
 		"upgradesTo": "Fighter",
 		"obsoleteTech": "Radar",
-		"uniques": ["[50]% chance to intercept air attacks", "[+150]% Strength <vs [Bomber] units>", "[+150]% Strength <vs [Helicopter] units>"],
+		"uniques": ["Can perform Air Sweep", "[50]% chance to intercept air attacks", "[+150]% Strength <vs [Bomber] units>", "[+150]% Strength <vs [Helicopter] units>"],
```

Comment:
Here Air Sweep is capitalized, but in the previous unique it is not?

Reply:
Capitalized all